lib/provider/camunda/CamundaPropertiesProvider.js

```diff
@@ -420,9 +421,22 @@ function createInputOutputTabGroups(element, bpmnFactory, elementTemplates, tran
 
   outputParameters(outputParametersGroup, element, bpmnFactory, elementTemplates, translate);
 
+  var errorsGroup = {
+    id: 'errors',
+    label: translate('Errors'),
+    entries: [],
+
+    enabled: function(element, node) {
+      return is(element, 'bpmn:ServiceTask') && getBusinessObject(element).type === 'external';
```

Comment:
I wonder if we would get safer if we use [internal helpers](https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/lib/helper/ImplementationTypeHelper.js#L137) for this.

```js
return ImplementationTypeHelper.getImplementationType(element) === 'external';
```

Reply:
Sure thing!